src/doc/trpl/installing-rust.md

```diff
@@ -25,6 +25,12 @@ $ sh rustup.sh
 [insecurity]: http://curlpipesh.tumblr.com
 
 If you're on Windows, please download the appropriate [installer][install-page].
+**NOTE:** By default, the Windows installer will not add Rust to the Path system
```

Comment:
Happy for it to be %PATH%.

----- Original Message -----
From: "rust-lang/rust" 
To:"rust-lang/rust" 
Cc:"Jonathan Hansford" 
Sent:Tue, 21 Jul 2015 08:52:27 -0700
Subject:Re: [rust] Guidance on Windows install re "Add to PATH"
(#27183)

	In src/doc/trpl/installing-rust.md [1]: 

	> @@ -25,6 +25,12 @@ $ sh rustup.sh > [insecurity]:
http://curlpipesh.tumblr.com > > If you're on Windows, please download
the appropriate [installer][install-page]. > +**NOTE:** By default,
the Windows installer will not add Rust to the Path system  

	Shouldn't this be %PATH% or something? 

	—
Reply to this email directly or view it on GitHub [2].     

Links:
------
[1] https://github.com/rust-lang/rust/pull/27183#discussion_r35118694
[2] https://github.com/rust-lang/rust/pull/27183/files#r35118694